pod2daemon/Makefile

```diff
@@ -66,7 +67,7 @@ bin/csi-driver-armv7: ARCH=armv7
 bin/csi-driver-ppc64le: ARCH=ppc64le
 bin/csi-driver-s390x: ARCH=s390x
 bin/csi-driver-%: $(SRC_FILES)
-	$(DOCKER_RUN) -e CGO_ENABLED=$(CGO_ENABLED) $(CALICO_BUILD) go build -v -o bin/csi-driver-$(ARCH) csidriver/main.go
+	$(DOCKER_RUN) -e CGO_ENABLED=$(CGO_ENABLED) $(CALICO_BUILD) go build -v $(RELEASE_FLAGS) -o bin/csi-driver-$(ARCH) csidriver/main.go
```

Comment:
In some cases you're doing this:

```
go build -v $(RELEASE_FLAGS) 
```

And in others (see kube-controllers) you have:

```
go build -v -ldflags "$(RELEASE_LDFLAGS) "
```

I think we need to make these consistent or else one of them won't work.

I'm thinking the pattern we should establish is like the one in the node repo.

At the top of the Makefile, we establish;

```
LDFLAGS = -ldflags "... $(RELEASE_LDFLAGS)"
```

Then in the build command we just do:

```
go build -v $LDFLAGS
```

WDYT?

Reply:
@caseydavenport Thanks for reviewing and for pointing me in the right direction. I did some changes based on your suggestions, could you please take a look and let me know if there is any problem?